src/components/hurricane-marker.scss

```diff
@@ -31,12 +31,20 @@
       margin-left: -22.5px;
       margin-top: -36px;
     }
+
+    img {
+      width: 200px;
+      height: 200px;
+      margin-left: -100px;
+      margin-top: -100px;
+    }
   }
 
+
   .categoryNumber {
-    color: #434343;
-    font-size: 16px;
-    text-shadow: 0 0 5px #fff;
+    color: #000;
+    font-size: 18px;
```

Comment:
Did you take a look at this with the track category markers? We may need to adjust the size of the circles.

Reply:
I checked Michael's spec and left your values, as they match it better.